arrow-site/doc/docs/arrow-core/datatypes/statet/README.md

```diff
@@ -315,7 +315,7 @@ fun main() {
 }
 ```
 
-While our code looks very similar to what we had before there are some key advantages. State management is now contained within `State` and we are dealing only with 1 monad instead of 2 nested monads so we can use monad bindings!
+While our code looks very similar to what we had before, there are some key advantages. State management is now contained within `State`, and we are dealing with only 1 monad instead of 2 nested monads, so we can use monad bindings!
```

Comment:
Numbers less than 10 should be written out: "one monad instead of two nested monads"